docsite/source/index.html.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Introduction
+layout: gem-single
+type: gem
+name: dry-operation
+sections:
+  - error-handling
+  - configuration
+  - extensions
+  - design-pattern
+---
+
+dry-operation provides an expressive and flexible way for you to model your app's business operations. It provides a lightweight DSL around [dry-monads](/gems/dry-monads/), which allows you to chain together steps and operations with a focus on the happy path, while elegantly handling failures.
```

Comment:
I tweaked this to start with a sentence that tries to describe the _user benefit_ first.

Reply:
Looks 👌